### PR TITLE
[docs] Add TNX AOT docs for Llama-2-70B

### PR DESCRIPTION
**Pull request by tosterberg**
## Description ##

Adding AOT doc for precompiling TNX models on EC2

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: serving/docs/lmi/tutorials/tnx_aot_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  --device /dev/neuron4 \
+  --device /dev/neuron5 \
+  --device /dev/neuron6 \
+  --device /dev/neuron7 \
+  --device /dev/neuron8 \
+  --device /dev/neuron9 \
+  --device /dev/neuron10 \
+  --device /dev/neuron11 \
+  763104351884.dkr.ecr.us-west-2.amazonaws.com/djl-inference:0.26.0-neuronx-sdk2.16.0 \
+  partition --model-dir /opt/ml/input/data/training --skip-copy
```
Comment (lanking520) [inline serving/docs/lmi/tutorials/tnx_aot_tutorial.md:112]: missing model preparation steps, where do we get llama 70b and where do we place it

Comment (tosterberg) [reply serving/docs/lmi/tutorials/tnx_aot_tutorial.md:156]: Added

File: serving/docs/lmi/tutorials/tnx_aot_tutorial.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+```
+
+We will run the `download.py` script defined above in order to download a local copy of the model into the `~/model` directory.
+
+```bash
+pip install huggingface_hub pathlib
+python3 download.py
+```
+
+After the download completes the `~/model` directory will contain the model artifacts that we can use for compilation.
```
Comment (lanking520) [inline serving/docs/lmi/tutorials/tnx_aot_tutorial.md:103]: in the below command, you didn't use this directory for mounting

Comment (tosterberg) [reply serving/docs/lmi/tutorials/tnx_aot_tutorial.md:103]: That is correct - the model folder attaches in this line.
  -v /home/ubuntu/:/opt/ml/input/data/training \  # Line 119
The model directory when this is followed verbatim ends up at /home/ubuntu/model, with the serving.properties in /home/ubuntu. The one thing I will have to double check is that you can just put model in the option.model_id.

**Review by lanking520 — APPROVED** [review]

**Pull request merged**